docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -0,0 +1,577 @@
+
+[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/WeatherForecastRuntimeIgnoreConverter.cs)]
+
+The converter causes the `Summary` property to be omitted from serialization if its value is null, an empty string, or "N/A". 
+
+Register this custom converter by [using an attribute](system-text-json-converters-how-to.md#registration-sample---jsonconverter-on-a-type) on the class or by [adding the converter](system-text-json-converters-how-to.md#registration-sample---converters-collection) to the <xref:System.Text.Json.JsonSerializerOptions.Converters> collection.
+
+This approach requires additional logic if:
+
+* The POCO includes complex properties.
+* You need to handle attributes such as `[JsonIgnore]` or options such as custom encoders.
+
+### Callbacks
+
+`Newtonsoft.Json` lets you execute custom code at several points in the serialization or deserialization process:
+
+* OnDeserializing (when beginning to deserialize an object)
+* OnDeserialized (when finished deserializing an object)
+* OnSerializing (when beginning to serialize an object)
+* OnSerialized (when finished serializing an object)
+
+In <xref:System.Text.Json>, you can simulate callbacks by writing a custom converter. The following example shows a custom converter for a POCO. The converter includes code that displays a message at each point that corresponds to a `Newtonsoft.Json` callback.
+
+[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/WeatherForecastCallbacksConverter.cs)]
+
+Register this custom converter by [using an attribute](system-text-json-converters-how-to.md#registration-sample---jsonconverter-on-a-type) on the class or by [adding the converter](system-text-json-converters-how-to.md#registration-sample---converters-collection) to the <xref:System.Text.Json.JsonSerializerOptions.Converters> collection.
+
+If you use a custom converter that follows the preceding sample:
+
+* The `OnDeserializing` code doesn't have access to the new POCO instance. To manipulate the new POCO instance at the start of deserialization, put that code in the POCO constructor.
+* Don't pass in the options object when recursively calling `Serialize` or `Deserialize`. The options object contains the `Converters` collection. If you pass it in to `Serialize` or `Deserialize`, the converter will be used, making an infinite loop that results in a stack overflow exception.
+
+## Scenarios that JsonSerializer currently doesn't support
+
+Workarounds are possible for the following scenarios, but some of them would be relatively difficult to implement. Code samples for workarounds are not available for these scenarios.
+
+If you implement a workaround for one of these scenarios and can share the code, select the button at the bottom of the page to send feedback for **This page**. That creates a GitHub issue and adds it to the issues that are listed at the bottom of the page.
```

Comment:
nit: lowercase?
```suggestion
If you implement a workaround for one of these scenarios and can share the code, select the button at the bottom of the page to send feedback for **this page**. That creates a GitHub issue and adds it to the issues that are listed at the bottom of the page.
```

Reply:
I'm copying the casing as shown on the button.